hibernate-core/src/main/java/org/hibernate/query/sqm/internal/DomainParameterXref.java

```diff
@@ -22,11 +22,13 @@
 import org.hibernate.query.sqm.tree.expression.JpaCriteriaParameter;
 import org.hibernate.query.sqm.tree.expression.SqmJpaCriteriaParameterWrapper;
 import org.hibernate.query.sqm.tree.expression.SqmParameter;
+import org.hibernate.type.BasicCollectionType;
 
 /**
  * Maintains a cross-reference between SqmParameter and QueryParameter references.
  *
  * @author Steve Ebersole
+ * @author Yanming Zhou
```

Comment:
Please don't change author tags for simple changes.  Thanks.

Reply:
done.